docs/origin_install.md

```diff
@@ -0,0 +1,154 @@
+
+        set resourcename = UFlorida-XRD
+
+### Configuring authentication and authorization
+
+XRootD offers several authentication options using security plugins to validate incoming credentials, such as bearer tokens, X.509 proxies, and VOMS proxies.
+Please follow the [XRootD authorization documentation](xrootd-authorization.md) for instructions on how to configure authentication and authorization,
+including validating credentials and mapping them to users if desired.
+
+### Optional configuration
+
+The following configuration steps are optional and will likely not be required for setting up a small site.
+If you do not need any of the following special configurations, skip to
+[the section on using XRootD](#using-xrootd).
+
+
+#### Enabling multi-user support
+
+!!! warning "Requirements for XRootD-Multiuser with VOMS FQANs"
+    Using XRootD-Multiuser with a VOMS FQAN requires mapping the FQAN to a username, which requires a `voms-mapfile`.
+    Support is available in `xrootd-voms 5.4.2-1.1`, in the OSG 3.6 repos, though it is expected in XRootD 5.5.0.
+    If you want to use multiuser, ensure you are getting `xrootd-voms` from the OSG repos.
+
+The `xrootd-multiuser` plugin allows XRootD to write files on the storage system as the
+[authenticated](xrootd-authorization.md) user instead of the `xrootd` user.
+If your XRootD service only allows read-only access, you should skip installation of this plugin.
+
+To set up XRootD in multi-user mode, install the `xrootd-multiuser` package:
+
+``` console
+root@xrootd-standalone # yum install xrootd-multiuser
+```
+
+Installing Pelican
+------------------
+
+Grab the appropriate binary for your system from the [pelican repository](https://github.com/PelicanPlatform/pelican/releases)
+
+Launching an Origin
+-------------------
+
+To launch a pelican origin, run:
+
+```./pelican origin -f <federation> -v <volume>```
+
+Where `<federation>` is the address of the federation the origin will be a part of and `<volume>` is the location where the data hosted on the origin will be mounted.
+
+The first time the origin is started, you will see something that looks like the following:
+
+[picture of an origin startup terminal prompt with the initializaiton code](origin_start.png)
+
+To initialize the admin interface (to see the metrics), go to the website specified (replace <> with `localhost`).
+
+You will see a warning that looks like the following (with some differences with respect to the browser):
+
+[invalid certificate warning image](invalid_certificate.png)
+
+Proceed despite the warning to get to the code entry page. Enter the code specified in the terminal and create a root metrics password:
+
+[code entry page](code_entry.png)
+
+[password entry page](password_entry.png)
+
+You should now see a webpage that looks like so:
+
+[metrics page](metrics_view.png)
+
+This will refresh every 10 minutes with the xrootd health metrics so that, as an admin, you can check the status of your origin.
```

Comment:
I'm not sure what we should say about registering the origin with the namespace registry. Eventually we want this to happen automatically. For now, the reality is that after serving the origin (which generates the asymmetric key pair needed by the registry) we also need to `pelican namespace register --prefix <namespace prefix> --namespace-url <registry url>`. This gives the registry the public key for the origin, and makes that public key available to all of the other services in the federation for verifying tokens.